proto/buf-generate.sh

```diff
@@ -1,3 +1,2 @@
 #!/bin/sh
-buf generate -o internal/proto buf.build/authzed/servok:4027ca77d1b04ea2888b00348c3fd3d9 --template $@
 buf generate -o internal/proto proto/internal --template $@
```

Comment:
If we're getting rid of the servok proto generation, you should delete its protos and you can get rid of this script entirely and just make the first line in buf.gen.yaml

```yaml
#!/usr/bin/env -S buf generate -o internal/proto proto/internal --template
```